airbyte-webapp-e2e-tests/cypress/plugins/index.ts

```diff
@@ -14,13 +14,25 @@
 
 import Cypress from "cypress";
 
+const pgp = require("pg-promise")();
+const postgresConfig = require(require("path").resolve("cypress.json"));
```

Comment:
This is just the `cypressConfig`?

Reply:
Oh, yeah, it's not obvious - will fix the naming! 👍